src/style.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  width: fit-content;
+}
```

it's not a list, so we don't need to reset the styles for it
```suggestion
.stars {
  display: flex;
}
```